cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -1018,6 +1018,61 @@ utxoStEx10 ::
   UTxOState era
 utxoStEx10 pf = UTxOState (utxoEx10 pf) (Coin 0) (Coin 5) def
 
+-- ====================================================================================
+--  Example 11: A transaction with multiple identicle certificates
```

Comment:
```suggestion
--  Example 11: A transaction with multiple identical certificates
```

Reply:
there were a few of these :grimacing: , thanks!